### ISSUE TITLE
Is there any way to move value?

### ISSUE THREAD

**Issue by stmalk** [toorshia/justgage#290]
I'm looking to change how the value looks. I.E. change its font size and move it a little up from where it is now. I couldn't find any setup parameters responsible for that, so is there any way to achieve that?

**Comment by richandhandsomeman** [comment]
Yes, I need this too and how do I add some small captions inside? For instance, I can use 'symbol' parameter, but I am not able to change its font size.
